### ISSUE TITLE
[suggestion] 

### ISSUE THREAD

**Issue by takecare** [ttencate/jfxr#42]

**Comment by ttencate** [comment]
Good idea! If you could send me a PR for the Electron config, I'd be happy to upload and maintain a download on itch.io. I can just copy the build and upload automation from another project, so it's hardly any extra work.

Caveat: if I do it like that there won't be a Mac version because I don't have a Mac to build it on. Blame Apple.

**Comment by takecare** [comment]
I haven't setup electron myself as I just used nativefier. Unfortunately I don't think it supports exporting the configuration it generates. I'll try to have a go when I have a bit more time though. As for the mac version, I don't mind building it myself and sending it over.

**Comment by wwderw** [comment]
I would think that it would be possible to integrate electron-builder into this repo here to build a version for all 3 platforms and set it up as releases on here.
